helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -274,7 +274,7 @@ starrocksCnSpec:
     #     scaleDown:
     #       selectPolicy: Disabled
   # define resources requests and limits for cn pods.
-  resource:
+  resources:
```

Comment:
could cause compatible issue when the user upgrade from previous chart version to this, without updating its values.yaml?

Reply:
yes, This is a problem. But this misspell should be fixed.
So, we both support resources and resource?

Reply:
yes, please. recommends the right spelling, but can still work if mis-spelled for now, possibly generating a warning message? can possibly remove the support after 2-3 releases in future.